apps/login-test-acceptance/setup/main.go

```diff
@@ -0,0 +1,3 @@
+package main
+
+func main() {}
```

Comment:
Why an empty main file?

Reply:
I know it's a bit ugly, but we build all the helpers using the same Dockerfile.
IMO we should use go also for the setup instead of bash.
Let's do that when we fix the acceptance tests.